.travis.yml

```diff
@@ -2,8 +2,6 @@ language: php
 
 php:
   - 5.3
-  - 5.4
-  - 5.5
```

Comment:
For speed up WIP removing less important scenarios. The commit has the title DON'T MERGE